uqcsbot/scripts/wolfram.py

```diff
@@ -0,0 +1,192 @@
+from uqcsbot import bot, Command
+from typing import Iterable, Tuple
+import requests
+import json
+
+APP_ID = "UVKV2V-5XW2TETT69"
+
+# IDEA: Show the assumptions made?
+# TODO: Better naming of commands
+@bot.on_command("wolframfull")
+async def handle_wolframfull(command: Command):
+    """
+    This posts the full results from wolfram query. Images and all
+
+    Example usage:
+    !wolframfull y = 2x + c
+    """
+    api_url = r"http://api.wolframalpha.com/v2/query?&output=json"
```

Comment:
why `r""`?

Reply:
Just a url habit of mine sometimes.

Reply:
I would remove the r unless its needed :^) just to be that little bit extra clear